projects/1.20-fabric/assets/a-man-with-plushies/a_man_with_plushies/lang/zh_cn.json

```
@@ -0,0 +1,317 @@
+{
+  "advancements.carrot_eater_pufferfish.descr": "给一只完全膨胀的河豚玩偶一根胡萝卜",
+  "advancements.carrot_eater_pufferfish.title": "河豚吃胡萝卜",
+  "advancements.first_common_plushie.descr": "获得一个爬行者玩偶",
+  "advancements.first_common_plushie.title": "可爱！",
+  "advancements.first_golden_plush_box.descr": "获得你的第一个金色毛绒盒子",
+  "advancements.first_golden_plush_box.title": "玩偶赌局！！！",
+  "advancements.first_plush_box.descr": "获得你的第一个毛绒盒子",
+  "advancements.first_plush_box.title": "玩偶！",
+  "advancements.first_rare_plushie.descr": "获得一个恶魂玩偶",
+  "advancements.first_rare_plushie.title": "绝非粗制滥造的珍品",
+  "advancements.first_spamton_voice_line.descr": "[[超链接屏蔽]]",
+  "advancements.first_spamton_voice_line.title": "现在是你成为大人物的机会！！",
```

补充：
https://deltarune.fandom.com/wiki/Spamton


```suggestion
  "advancements.first_spamton_voice_line.title": "现在是你成为[[大人物]]的机会！！",
```

不知道deltarune简中是怎么翻的，建议参考那边